src/data/update.ts

```diff
@@ -24,12 +24,13 @@ export enum UpdateEntityFeature {
 interface UpdateEntityAttributes extends HassEntityAttributeBase {
   auto_update: boolean | null;
   installed_version: string | null;
-  in_progress: boolean | number;
+  in_progress: boolean;
   latest_version: string | null;
   release_summary: string | null;
   release_url: string | null;
   skipped_version: string | null;
   title: string | null;
+  update_percentage: number | null;
```

Comment:
Will this attribute always be around, even when not updating of not supported?

Reply:
Yes, that's how it's implemented in the core PR